components/compositing/webview.rs

```diff
@@ -0,0 +1,188 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use std::collections::HashMap;
+
+use msg::constellation_msg::TopLevelBrowsingContextId;
+
+#[derive(Debug)]
+pub struct WebViewManager<WebView> {
+    /// Our top-level browsing contexts. In the WebRender scene, their pipelines are the children of
+    /// a single root pipeline that also applies any pinch zoom transformation.
+    webviews: HashMap<TopLevelBrowsingContextId, WebView>,
+
+    /// The order to paint them in, topmost last.
+    painting_order: Vec<TopLevelBrowsingContextId>,
+}
+
+impl<WebView> Default for WebViewManager<WebView> {
+    fn default() -> Self {
+        Self {
+            webviews: Default::default(),
+            painting_order: Default::default(),
+        }
+    }
+}
```

Comment:
I think you can just derive from `Default` for this?

Reply:
Fixed in [8c01639](https://github.com/servo/servo/pull/31417/commits/8c01639b1993c3b6c5fa57c1fcfc412716cf6f05)